src/main/java/fi/otavanopisto/kuntaapi/server/rest/OrganizationsApiImpl.java

```diff
@@ -648,7 +649,8 @@ public Response findOrganizationSetting(String organizationIdParam, String setti
   @Override
   @SuppressWarnings ("squid:MethodCyclomaticComplexity")
   public Response updateOrganizationSetting(String organizationIdParam, String settingId, OrganizationSetting setting) {
-    OrganizationId organizationId = toOrganizationId(organizationIdParam);
+    return createNotImplemented(NOT_IMPLEMENTED);
+    /*OrganizationId organizationId = toOrganizationId(organizationIdParam);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)